Freshli/SemVerVersionInfo.cs

```diff
@@ -32,7 +36,7 @@ public class VersionInfo : IComparable {
     public string BuildMetadata { get; private set; }
 
     private readonly Regex _versionExpression = new Regex(
-      @"^v?(\d+)[\._]?(\d+)?[\._]?(\d+)?" +
+      @"^v?V?(\d+)[\._]?(\d+)?[\._]?(\d+)?" +
```

Comment:
Added to support https://packagist.org/packages/barryvdh/laravel-debugbar V2.2.3